test/wolff_dynamic_topics_tests.erl

```diff
@@ -0,0 +1,135 @@
+-module(wolff_dynamic_topics_tests).
+
+-include("wolff.hrl").
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("kafka_protocol/include/kpro.hrl").
+
+-define(KEY, key(?FUNCTION_NAME)).
+-define(HOSTS, [{"localhost", 9092}]).
+
+%% Checks that having different producers to the same kafka topic works.
```

Comment:
Nit: the test doesn't seem to actually test 2 different dynamic producer groups targeting the same topic(s).